jsonschema-core.xml

```diff
@@ -1830,6 +1830,48 @@
                 </section>
             </section>
 
+            <section title="Compound Documents">
+                <t>
+                    A Compound Schema Document is defined as JSON document (sometimes called a "dereferenced" or
+                    "bundled" schema) which has multiple embedded JSON Schema Resources bundled into the same
+                    document to ease transportation.
+                </t>
+                <t>
+                    Each embedded Schema Resource MUST be treated as an individual Schema Resource, following standard
+                    schema loading and processing requirements, including determining voacbulary support.
+                </t>
+                <section title="Bundling">
+                    <t>
+                        A Compound Schema Document is created by replacing references (such as "$ref") with an
+                        embedded version of the referenced Schema Resource.
```

Comment:
I feel like we might need a distinction between references to other documents vs references within the same document.  Because really bundling means getting all of the resources into one document.  It may or may not be possible (or desirable) to dereference things.

"Bundling" could simply mean "stuff each referenced resource that is in another document under this document's `$defs`.  This doesn't even require changing any of the references since they are targeting by `$id` and that doesn't change through this basic form of bundling.

Reply:
My upcoming update to the PR suggests...
```
                        A Compound Schema Document is created by taking references (such as "$ref")
                        to an external Schema Resource and embedding the referenced Schema Resources 
                        within the referring document.
```

Does that sound enough?

Reply:
@Relequestual yes that sounds great!